src/shipit/frontend/src/views/ListReleases/index.js

```diff
@@ -530,7 +543,7 @@ class TaskLabel extends React.PureComponent {
                   <Button
                     onClick={this.doEet}
                     bsStyle="danger"
-                    disabled={!this.context.authController.userSession && !this.state.submitted}
+                    disabled={!this.context.authController.userSession || this.state.inProgress}
```

Comment:
I matched the logic in `NewRelease/index.js` which does `|| this.state.inProgress` instead of `&& !this.state.submitted`